src/modules/core.js

```diff
@@ -1679,17 +1679,17 @@
 			// delete existing biography banner. when accepting, reviewer is forced to choose if it's a biography or not, so we'll add (or not add) our own biography banner later
 			banners = banners.filter( ( value ) => !value.match( /^{{WikiProject Biography/i ) );
 
-			// add biography banner to array. and add |living= and |listas= to banner shell
+			// add biography banner to array. and add |blp= and |listas= to banner shell
 			let bannerShellExtraParams = '';
 			if ( isBiography ) {
 				banners.push(
 					'{{WikiProject Biography}}'
 				);
 
-				if ( lifeStatus === 'living' ) {
-					bannerShellExtraParams += ' |living=yes';
+				if ( lifeStatus === 'blp' ) {
```

Comment:
Internally in AFCH (but not in the wikicode), can we keep using "living" as the value? I find living to be more readable.